packages/wrangler/src/pages.tsx

```diff
@@ -1098,29 +1101,77 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
       )
     )
     .command("project", false, (yargs) =>
-      yargs.command(
-        "list",
-        "List your Cloudflare Pages projects",
-        () => {},
-        async (args) => {
-          const config = readConfig(args.config as ConfigPath, args);
-          const accountId = await requireAuth(config);
-
-          const projects: Array<Project> = await listProjects({ accountId });
+      yargs
+        .command(
+          "list",
+          "List your Cloudflare Pages projects",
+          () => {},
+          async (args) => {
+            const config = readConfig(args.config as ConfigPath, args);
+            const accountId = await requireAuth(config);
+
+            const projects: Array<Project> = await listProjects({ accountId });
+
+            const data = projects.map((project) => {
+              return {
+                "Project Name": project.name,
+                "Project Domains": `${project.domains.join(", ")}`,
+                "Git Provider": project.source ? "Yes" : "No",
+                "Last Modified": project.latest_deployment
+                  ? timeagoFormat(project.latest_deployment.modified_on)
+                  : timeagoFormat(project.created_on),
+              };
+            });
+            render(<Table data={data}></Table>);
+          }
+        )
+        .command(
+          "create [name]",
+          "Create a new Cloudflare Pages project",
+          (yargs) =>
+            yargs
+              .positional("name", {
+                type: "string",
+                demandOption: true,
+                description: "The name of your Pages project",
+              })
+              .options({
+                "production-branch": {
+                  type: "string",
+                  // TODO: Should we default to the current git branch?
+                  default: "production",
```

Comment:
I think we should default to `main` or `master` unless this is passed in

Reply:
I think to keep aligned with industry ethical/best practice `main` should be encouraged when possible.

Reply:
We're using `production` elsewhere, so we'll stick with that. This isn't necessarily a "branch" in the typical git sense—it's more of an "environment name".